### PR TITLE
Add per-page layout

### PR DESCRIPTION
**Pull request by AbdBarho**
This PR builds on the discussion in #76 

Some layout fixes are necessary, mainly removing the `h-full mx-auto`.

### CODE REVIEW

**Comment by fozziethebeat** [comment]
I'll review this after my PR finally goes in.

File: website/src/components/Layout.tsx
Comment (fozziethebeat) [inline website/src/components/Layout.tsx:12]: This component is pretty magical.  Can you add some high level comments on how it works.  My understanding is that any new page component should set a `getLayout` field and use this method.  That'll use this one unified layout.

Should we name this as something a little more specific like `getDefaultLayout` in the situation where we have another layout?

**Review by fozziethebeat — COMMENTED** [review]
Looks good.  Will approve after you've merged in my PR changes.

File: website/src/components/Layout.tsx
Comment (AbdBarho) [reply website/src/components/Layout.tsx:12]: You are totally right, I changed it now so it is the default layout (you don't have to specify anything in the components), with the option of overriding the layout if needed.

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**
